<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>80</y>
     <width>711</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>30</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCamera">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera On</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRecord">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Rec Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCapture">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenVideo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenVideo_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>660</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
